tools/component-builder-simple/index.js

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const gulp = require('gulp');
+const rename = require('gulp-rename');
+const del = require('del');
+const css = require('./css');
+
+function clean() {
+  return del('dist/*');
+}
+
+const build = gulp.series(
+  clean,
+  css.buildCSS,
+  function copyIndex() {
+    // Just copy index.vars as index.css to maintain backwards compat
+    return gulp.src('dist/index.css')
+      .pipe(rename((file) => {
+        file.basename = 'index-vars';
+      }))
+      .pipe(gulp.dest('dist/'))
+  }
+);
+
+exports.default = build;
+exports.build = build;
+exports.buildLite = build;
+exports.buildMedium = build;
+exports.buildHeavy = build;
```

Comment:
We may way to make `buildLite` avoid checking CSS for undefined properties so `gulp dev` works even when things are broken. The advantage is that you can debug live, the disadvantage is you don't see that bug until you try to build for release (full build).


Right now, `gulp dev` will break if there are any unused or undefined variables.